src/resolver.ts

```diff
@@ -1,15 +1,30 @@
 import { UserInput } from './UserInput';
 import { User } from './entity/User';
 import { AppDataSource } from './data-source';
+import { CustomError } from './custom-errror';
+import { Md5 } from 'ts-md5';
 
 export const resolvers = {
   Mutation: {
     createUser: async (_, args: { data: UserInput }) => {
       const { data } = args;
+      if (data.password.length < 6) {
+        throw new CustomError('A senha deve conter pelo menos 6 caracteres', 400);
+      }
+      if (!thereIsLetter(data.password)) {
+        throw new CustomError('A senha deve conter pelo menos 1 letra', 400);
+      }
+      if (thereIsNumber(data.password)) {
```

Comment:
```suggestion
      if (!isThereANumber(data.password)) {
```

Precisava negar aqui tbm pra jogar erro né? Pq se tiver um número ele não deveria entrar no if, certo?